contracts/deployments/bridge/v0.2.2.json

```diff
@@ -0,0 +1,13 @@
+{
+  "metadata": {
+    "version": "v0.2.2",
+    "date": "25-04-2025",
+    "by": "Evan",
+    "from_machine": "Evan's Macbook",
+    "from_branch": "canary",
+    "from_tag": "v0.2.1"
+  },
+  "addresses": {
```

Comment:
Can we somehow write where the rest of addresses can be found?

Reply:
done